### ISSUE TITLE
Canvas aspect ratio incorrect

### ISSUE THREAD

**Issue by JVital2013** [adizanni/floor3d-card#27]
**Checklist:**

- [x] I updated to the latest version available
- [x] I cleared the cache of my browser

**Release with the issue:**
0.9.18

**Last working release (if known):**
0.9.14

**Browser and Operating System:**
Chrome on Windows and Mac; Safari on iPhone

**Description of problem:**

When the card is in panel mode, the canvas does not render with the correct aspect ratio. It seems to be expecting the aspect ratio of the Card view, but the ratio can vary wildly in panel mode. Depending on the browser size this is OK, but it looks particularly bad on phones: the canvas only takes up the top part of the screen and is very small. This issue did not exist in 0.9.14.

Example: [https://imgur.com/a/GRENHr4](https://imgur.com/a/GRENHr4)

**Javascript errors shown in the web inspector (if applicable):**

```
None
```

**Additional information:**

**Comment by adizanni** [comment]
Hello Jvital2013, as a temporary workaround, please put a style in the parameter like this: 

```yaml
style: 'height: 100vh'
```

**Comment by JVital2013** [comment]
Ah, I didn't even think about messing with the style attribute. Thanks, that makes for a great patch until it gets properly fixed! FWIW, this seems to work a bit better cross-platform for me:

```
style: 'height: 100%'
```

I want to take this opportunity to thank for your hard work on this card. I killed a whole weekend measuring my entire house and building it Sweet Home 3D, just so I can implement this card. It was a frivolous endeavor in the grand scheme of things, but the end result is SOOO COOOL!!!

Floor3D-Card really is a well-implemented, innovative project  - it's what makes open source, modular software like HomeAssistant great. Keep it up!

**Comment by adizanni** [comment]
I hope I can keep up with the expectation, because the card is becoming complex and I do not have all needed skills to make it up to high standards. I will see going forward if there are some volunteer to take it over or to do a team work......

**Comment by JVital2013** [comment]
I fully understand. I've gotten a bit over my head with some projects before... my RoadApplePi repo is a great example of that. I got it to a point, then I burned out on it. But hey, at least it's buried in the Arctic. That's cool I guess... alas, I'm getting off topic.

I'll see if I can fix this bug - but no promises :). I've never messed with `npm build` before, but it doesn't look too hard. I might be able to do something about _resizeCanvas() too. It might be a week or two before I get to it though
